types/proposal.go

```diff
@@ -23,25 +23,27 @@ var (
 // a so-called Proof-of-Lock (POL) round, as noted in the POLRound.
 // If POLRound >= 0, then BlockID corresponds to the block that is locked in POLRound.
 type Proposal struct {
-	Type      tmproto.SignedMsgType
-	Height    int64     `json:"height"`
-	Round     int32     `json:"round"`     // there can not be greater than 2_147_483_647 rounds
-	POLRound  int32     `json:"pol_round"` // -1 if null.
-	BlockID   BlockID   `json:"block_id"`
-	Timestamp time.Time `json:"timestamp"`
-	Signature []byte    `json:"signature"`
+	Type       			  tmproto.SignedMsgType
+	Height                int64     `json:"height"`
+	CoreChainLockedHeight uint32    `json:"coreHeight"`
```

Comment:
`coreChainLockedHeight`?

Reply:
For JSON I do not think we should have such a long string, it seems wasteful. However it should be snake_case I think.

Reply:
I'm ok to keep it short. But you right, it must be a snake case.

BTW, I feel like it's better to add our fields to the end of structures so it will be easier to merge with upstream and see changes from our side. Any thoughts?